src/app/static/src/tests/components/plots/utils.test.ts

```diff
@@ -428,4 +428,42 @@ describe("plot utils", () => {
 
         expect(result).toStrictEqual([12, 0.5, 14, 0.6, 14]);
     });
+
+    it("findPath produces expected result", () => {
```

Comment:
It's a good idea with functions like this to unit test multiple cases, including edge/unusual cases if there are any - here I'd add a test where the first argument, the id, comes from each level. So test it brings back the right thing for `MWI` and `MWI11` say.